card/db/bootstrap/card_actions.yml

```diff
@@ -2933,1269 +2933,1514 @@ card_actions_419:
   action_type: 0
   draft: false
```

Comment:
starting to think we should put the bootstrap data in a subrepo and maybe a separate gem.  really becoming a nuisance for code review :(